tests/resetDB.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const Knex = require("knex");
+
+const AppConstants = require("../app-constants");
+const DB = require("../db/DB");
+const knexConfig = require("../db/knexfile");
+const {seed} = require("../db/seeds/test_subscribers");
+
+
+// (Re-)create DB connection at the beginning of each test suite
+beforeAll(async () => {
+  DB.createConnection();
+});
+
+
+beforeEach(async () => {
+  const knex = Knex(knexConfig[AppConstants.NODE_ENV]);
+  await knex("subscribers").truncate();
+  await seed(knex);
+  knex.destroy();
+});
```

Comment:
Should we have some `DB.reset()` method, or would that be too risky?

Reply:
Yeah, I thought it would be too risky.